pypi_formula_mappings.json

```diff
@@ -139,7 +139,8 @@
     "exclude_packages": ["pillow"]
   },
   "openstackclient": {
-    "extra_packages": ["python-heatclient", "python-octaviaclient"]
+    "extra_packages": ["python-heatclient", "python-octaviaclient"],
+    "exclude_packages": ["pillow"]
```

Comment:
Should we add `depends_on "pillow"` or do we just want to skip pillow dependency?
Also, probably, we want to add `six` here?

Reply:
That was a typo, I had meant to do `"exclude_packages": ["six"]`. Good catch, I've fixed it now.